docs/troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting
```

Comment:
can you please reference this doc in the main `README.md`?

Reply:
+1